File: qasdk/src/main/java/co/astrnt/qasdk/type/MediaTypes.java
package co.astrnt.qasdk.type;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import androidx.annotation.StringDef;

import static co.astrnt.qasdk.type.MediaType.AUDIO;
import static co.astrnt.qasdk.type.MediaType.VIDEO;


@StringDef({VIDEO, AUDIO})
@Retention(RetentionPolicy.SOURCE)
public @interface MediaTypes {
}

File: qasdk/src/main/java/co/astrnt/qasdk/type/MediaTypes.kt
package co.astrnt.qasdk.type

import androidx.annotation.StringDef
import co.astrnt.qasdk.type.MediaType.AUDIO
import co.astrnt.qasdk.type.MediaType.VIDEO

@StringDef(VIDEO, AUDIO)
@kotlin.annotation.Retention(AnnotationRetention.SOURCE)
annotation class MediaTypes 